### PR TITLE
Translate features

### PR DESCRIPTION
**Pull request by huntzhan**

### CODE REVIEW

File: website/docs/feature/geometric-distortion/affine.md [hunk 1]
```diff
@@ -13,30 +19,29 @@ from vkit.augmentation.geometric_distortion import (
...
 ```
 
-其中：
+Parameters:
 
-* `angle`：取值范围 `(-90, 90)`，正数为向右剪切角度，负数向左
+* `angle`: The shearing angle in range `(-90, 90)`. Positive value for rightward shearing
 
-效果示例：
+## shear_vert
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/affine.md:35]: Missing the demo gif in the English version

File: website/docs/feature/geometric-distortion/affine.md [hunk 2]
```diff
@@ -45,30 +50,29 @@ from vkit.augmentation.geometric_distortion import (
...
 ```
 
-其中：
+Parameters:
 
-* `angle`：取值范围 `(-90, 90)`，正数为向下剪切角度，负数向上
+* `angle`: The shearing angle in range `(-90, 90)`. Positive value for downward shearing
 
-效果示例：
+## rotate
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/affine.md:66]: Similarly missing demo gif in the English version

File: website/docs/feature/geometric-distortion/affine.md (hunk 1 above)
Comment (huntzhan) [reply website/docs/feature/geometric-distortion/affine.md:35]: Try to reload your page.

File: website/docs/feature/geometric-distortion/affine.md [hunk 3]
```diff
@@ -109,32 +112,29 @@ from vkit.augmentation.geometric_distortion import (
...
-    # [0.0, 1.0), shrink the right side.
-    # The larger abs(ratio), the more to shrink.
     ratio: float
 ```
 
-其中：
+Parameters:
 
-* `ratio`：表示纵向缩减比例，取值范围 `(-1.0, 1.0)`，正数缩减右边，负数缩减左边，绝对值越大缩减的量越大，倾斜效果越明显
+* `ratio`: The ratio to shew, with value in range `(-1.0, 1.0)`. Positive value for skewing the right side, while negative value for skewing the left side. The larger value, the more to skew
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/affine.md:125]: * `ratio`: The ratio to shew, with value in range `(-1.0, 1.0)`. Positive value for skewing to the right, while negative value for skewing to the left. The degree of skewness increases with the absolute value of `ratio`

File: website/docs/feature/geometric-distortion/affine.md [hunk 4]
```diff
@@ -143,24 +143,14 @@ from vkit.augmentation.geometric_distortion import (
...
-* `ratio`：表示横向缩减比例，取值范围 `(-1.0, 1.0)`，正数缩减下边，负数缩减上边，绝对值越大缩减的量越大，倾斜效果越明显
-
-效果示例：
-
-<div align="center">
-    <img alt="skew_vert.gif" src="https://i.loli.net/2021/11/28/V9cOmJZuRLXlk8r.gif" />
-</div>
+Parameters:
 
+* `ratio`: The ratio to shew, with value in range `(-1.0, 1.0)`. Positive value for skewing the top side, while negative value for skewing the bottom side. The larger value, the more to skew
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/affine.md:156]: * `ratio`: The ratio to shew, with value in range `(-1.0, 1.0)`. Positive value for skewing to the top, while negative value for skewing to the bottom. The degree of skewness increases with the absolute value of `ratio`

File: website/docs/feature/geometric-distortion/camera.md [hunk 5]
```diff
@@ -1,10 +1,25 @@
...
+Description: Simulate 3D surface curving effects based on camera model and cubic spline, as described in [Page dewarping](https://mzucker.github.io/2016/08/15/page-dewarping.html)  
 
-import:
+Effect:
+
+<div align="center">
+    <img alt="camera_cubic_curve.gif" src="https://i.loli.net/2021/11/25/B7Rpz46u5axO1sf.gif" />
+</div>
+
+Among the animation:
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/camera.md:13]: In the above animation:

File: website/docs/feature/geometric-distortion/camera.md [hunk 6]
```diff
@@ -30,50 +45,39 @@ class CameraModelConfig:
...
 
-* `CameraModelConfig` 是相机模型的配置（下同）
-  * `rotation_unit_vec`：旋转单位向量，即旋转指向方向，具体见 [cv.Rodrigues](https://docs.opencv.org/4.5.3/d9/d0c/group__calib3d.html#ga61585db663d9da06b68e70cfbf6a1eac) 里的解释
-  * `rotation_theta`：旋转的角度，区间 `[-180, 180]`。角度为正数，表示的方向与右手法则旋转方向一致
-  * `principal_point`：可选。相机成像光学轴（optical axis）与图片的相交点，使用原图的坐标表示。如果不提供，默认会设定为图片的中心点
-  * `focal_length`：可选。相机成像光学轴的焦距。如果不提供，会采用图像长宽中较大值作为焦距
-  * `camera_distance`：可选。指定相机坐标原点到 `principal_point` 的距离。如果不提供，会基于图片与成像屏幕相切的策略决定此距离
+* `rotation_unit_vec`: An unit rotation vector, as explained in [cv.Rodrigues](https://docs.opencv.org/4.5.3/d9/d0c/group__calib3d.html#ga61585db663d9da06b68e70cfbf6a1eac) 
+* `rotation_theta`: The angle of rotation,  accepting degrees in range  `[-180, 180]`. Positive value corresponds to the rotation direction following the right-hand rule
+* `principal_point`: Optional. The intersection point of the optical axis and the image, described in the image 2D coordinate. If not provided, the center of image will be selected as the principal point
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/camera.md:58]: * `principal_point`: Optional. The intersection point of the optical axis and the image, described in 2D coordinate with regards to the original image. If not provided, the center of image will be selected as the principal point

File: website/docs/feature/geometric-distortion/camera.md [hunk 7]
```diff
@@ -83,37 +87,37 @@ from vkit.augmentation.geometric_distortion import (
...
 
-其中：
+Parameters:
 
-* `fold_point`  与  `fold_direction`  决定基准线。 `fold_point`  设为原图的某个点，`fold_direction` 为从该点出发的基准线角度，顺时针区值区间 `[0, 180]`
-* `fold_perturb_vec`：为三维扰动向量。图中的点与基准线越接近，扰动越强，即 `p + w * fold_perturb_vec`
-* `fold_alpha`： 控制 `w = fold_alpha / (fold_alpha + d)`，`d` 为点到翻折线的归一化距离。`fold_alpha` 的取值越靠近 `0`，翻折效果越强。推荐取值 `0.5`
+* `fold_point`  and  `fold_direction`  define a standard line.  `fold_point` is a point in the image 2D coordinate.  `fold_direction`  is the clockwise angle of line through `fold_point` , with the value in range `[0, 180]` 
+* `fold_perturb_vec`:  The perturbed vector, defining the direction and strength of the distortion. For each position in the original image,  the weight of perturbed vector will be generated based on the distance to the standard line. And the distorted position is generated by the formula `p + w * fold_perturb_vec`, where `p` represents the original position   
+* `fold_alpha`: Control the weight `w = fold_alpha / (fold_alpha + d)`, where `d` represents the the normalized distance to the standard line. The smaller `fold_alpha` (closer to `0`), the stronger folding effect. Recommand: `0.5`
```
Comment (Yuri-x) [inline website/docs/feature/geometric-distortion/camera.md:107]: Recommand -> Recommended value

File: website/docs/feature/photometric-distortion/color.md [hunk 8]
```diff
@@ -75,29 +82,31 @@ from vkit.augmentation.photometric_distortion import (
...
+Parameters:
+
+* `rnd_state`: Optional. If provided, should be the same type as `numpy.random.RandomState.get_state()`. `rnd_state` is used to initialize `numpy.random.RandomState` to control the randomness
+
 
-* `rnd_state`: 可选，类型与  `numpy.random.RandomState.get_state()` 的返回值一致，用于初始化 `numpy.random.RandomState`。默认情况会随机初始化
+## `hue_shift`
 
-效果示例：
+Description: Shift the mean of hue channel. Notice the input `VImage` should have `HSV` mode
```
Comment (Yuri-x) [inline website/docs/feature/photometric-distortion/color.md:100]: Notice -> Note that

**Comment by huntzhan** [comment]
LGTM

**Pull request merged**